docs/toc.yml

```diff
@@ -39,3 +39,6 @@ entries:
 
 - file: tutorials/FlowTutorial
   title: Flow Tutorial
```

Comment:
Can you create a separate tutorial for the Colab based execution of GCD under Tutorials? The title says Flow Tutorial. The title should be something like "Running the OpenROAD flow in Google Colab"